cobalt/browser/performance/performance_impl.cc

```diff
@@ -35,4 +35,9 @@ void PerformanceImpl::MeasureAvailableCpuMemory(
   std::move(callback).Run(base::SysInfo::AmountOfAvailablePhysicalMemory());
 }
 
+void PerformanceImpl::MeasureUsedCpuMemory(
+    MeasureAvailableCpuMemoryCallback callback) {
+  std::move(callback).Run(base::SysInfo::AmountOfPhysicalMemory());
```

Comment:
QQ: Shouldn't this be `AmountOfPhysicalMemory() - AmountOfAvailablePhysicalMemory()` ?

Reply:
Totally right, fixed